lib/ts/types.ts

```diff
@@ -116,7 +116,7 @@ export type RecipeModuleHooks = {
     /*
      * Optional method used for redirections.
      */
-    getRedirectionURL?: (context: { action: string }) => Promise<string | undefined>;
+    getRedirectionURL?: (context: { action: string; redirectToPath?: string }) => Promise<string | undefined>;
```

Comment:
I don't like how recipe module is typed either but since it's generic. We have no idea if action is "SUCCESS", "ERROR", etc... it depends on the recipe. That's one reason I wasn't sure we should have `redirectToPath` as a query params, we can discuss to try to find a better solution.